spec/abci/abci++_app_requirements.md

```diff
@@ -276,20 +276,6 @@ time.
 
 The existence of this global mutex means Go application developers can get thread safety for application state by routing all reads and writes through the ABCI system. Thus it may be unsafe to expose application state directly to an RPC interface, and unless explicit measures are taken, all queries should be routed through the ABCI Query method.
 
-<!--
```

Comment:
For `v0.38.x` this is good.
However, for `main` and `v1.x` you may want to explain here the new unsync client (check [here)](https://github.com/cometbft/cometbft/blob/v1.0.0-alpha.2/abci/client/unsync_local_client.go#L32) we now have. You would need to adapt the commented text: in `v0.36.x` the former Tendermint Core team decided to remove the mutex altogether, whereas `v1.x` gives the flexibility of keeping the mutex or not, but providing the extra ABCI client.

Reply:
I can open a separate issue for this on v1, thanks for pointing that out

Reply:
Thanks